storageClient/storageClient.go

```diff
@@ -45,6 +45,17 @@ func (c *StorageClient) WriteNote(ctx context.Context, params Note) (*Note, erro
 	return result, err
 }
 
+func (c *StorageClient) PostGroups(ctx context.Context, params Group) (*Group, error) {
```

Comment:
We should have a more functional name like "CreateGroup" and accept the different param/response types as mentioned above